Resources/doc/cookbook/recipe_knp_menu.rst

```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
 
 Please see the `Using events to allow a menu to be extended`_ for further information.
 
+
+Hide menu item
+------------------
+
+You can modify the menu to hide some menu item. You need to add option 'show_in_dashboard' in your admin services or simple remove menu item from sonata_admin dashboard group configuration:
```

Comment:
@greg0ire I mean why I need to do it? I don't find any lines with linebreak in this document. This document contains a line much more longer. Is it some new rule about line length?